src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclint/DocLint.java

```diff
@@ -346,12 +343,9 @@ private void processArgs(Env env, String... args) {
             } else if (arg.startsWith(XCUSTOM_TAGS_PREFIX)) {
                 env.setCustomTags(arg.substring(arg.indexOf(":") + 1));
             } else if (arg.startsWith(XHTML_VERSION_PREFIX)) {
-                String argsVersion = arg.substring(arg.indexOf(":") + 1);
-                HtmlVersion htmlVersion = HtmlVersion.getHtmlVersion(argsVersion);
-                if (htmlVersion != null) {
-                    env.setHtmlVersion(htmlVersion);
-                } else {
-                    throw new IllegalArgumentException(argsVersion);
+                String argVersion = arg.substring(arg.indexOf(":") + 1);
+                if (argVersion == null || !argVersion.equals("html5")) {
+                    throw new IllegalArgumentException(argVersion);
```

Comment:
These lines are likely to be used as long as the "--doclint-format html5" option is permitted.  For example, this option is still used in the make/common/JavaCompilation.gmk.

Reply:
1. we can update make/common/JavaCompilation.gmk
2. the option in javac/javadoc should be converted to a no-op so that it does not get passed down to doclint

Reply:
(updating make/common/JavaCompilation.gmk should be done with a separate JBS issue, filed against the build.)

Reply:
Filed as [JDK-8256313](https://bugs.openjdk.java.net/browse/JDK-8256313).

Reply:
In javac Arguments class, we should delete these lines at about 855

````
        String format = options.get(Option.DOCLINT_FORMAT);
        if (format != null) {
            doclintOpts.add(DocLint.XHTML_VERSION_PREFIX + format);
        }
````

Reply:
Filed as [JDK-8257204](https://bugs.openjdk.java.net/browse/JDK-8257204).